README.md

```diff
@@ -8,12 +10,16 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/perses/perses)](https://goreportcard.com/report/github.com/perses/perses)
 [![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/perses/perses)
 
+</div>
+
 ## Overview
 
 Perses is part of the [CoreDash community](https://github.com/coredashio/community). It belongs to the Linux Foundation.
 At a later stage, we want to promote the project to the [Cloud Native Computing Foundation](https://www.cncf.io/) and be
 part of the monitoring tools like Prometheus or Thanos.
 
+![img.png](docs/images/node_exporter_full_dashboard.png)
```

Comment:
could the sample graph not have a datasource=grafana variable ? :D
\+ dark theme maybe ?

Reply:
Oops 🙈